modules/plugins/utility/motion/precognition/default.nix

```
@@ -0,0 +1,6 @@
+_: {
```

```suggestion
{
```

Function argument can be omitted if empty.

---

Resolved in b0641a8c0b4e1384af0c2017ca659883befa93d2.